code/modules/mob/living/carbon/carbon.dm

```diff
@@ -244,6 +244,13 @@
 
 		paper_note.show_through_camera(usr)
 
+/mob/living/carbon/on_fall()
+	. = ..()
+	loc?.handle_fall(src)
+	SHOULD_CALL_PARENT(TRUE)
+	SEND_SIGNAL(src, COMSIG_LIVING_THUD)
+	loc?.handle_fall(src) //it's loc so it doesn't call the mob's handle_fall which does nothing
```

Comment:
You copy pasted all the code instead of just copy pasting the comment

Reply:
deatgggggggggg